user_management/src/user_management/JWTManager.py

```diff
@@ -9,7 +9,7 @@
 def user_exist(user_id: int) -> bool:
     if (user_id is None
             or user_id == ''
-            or not isinstance(user_id, int)
+            or type(user_id) is not int
```

Comment:
Why don't use isinstance ? I think that isinstance is more modern

Reply:
Because I did not the existence of isinstance.
In this case, type() does the job, it's not obsolete, I don't think there are any [subclass problems](https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance#1549814), so I'd like to keep it this way